runner/components/io.siddhi.distribution.metrics.core/src/main/java/io/siddhi/distribution/metrics/prometheus/reporter/impl/PrometheusReporter.java

```diff
@@ -64,10 +72,228 @@ public void startReporter() {
                 .convertDurationsTo(TimeUnit.MILLISECONDS)
                 .build();
 
+        collectorRegistry = new CollectorRegistry();
+
         try {
             URL target = new URL(serverURL);
-            collectorRegistry = new CollectorRegistry();
-            collectorRegistry.register(new DropwizardExports(metricRegistry));
+
+//          Configuration for metrics without functions.
+            MapperConfig queryMetricConfig = new MapperConfig();
+            queryMetricConfig.setMatch("io.siddhi.SiddhiApps.*.Siddhi.Queries.*.*");
+            queryMetricConfig.setName("siddhi.query" + "." + "${2}");
+            Map<String, String> queryMetricLabels = new HashMap<String, String>();
```

Comment:
It seems these are constants, any reason to not have them in a resource YAML file? IMO it will be easier to maintain

Reply:
The above requested change is addressed by the commit 2523d00.